spec/models/history_entry_spec.rb

```diff
@@ -129,6 +129,14 @@
       s._id.in?(res).should be_falsy
     end
 
+    it "should get value at limit dates" do
+      s = Contact.make(level: 'aspirante')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.